packages/client/src/features/modal/components/index.ts

```diff
@@ -0,0 +1,3 @@
+import CancelBookingDialog from "./CancelBookingDialog";
```

Comment:
I expected to find the definition of `componentWhitelist` here but it's not here. What am I missing?

Reply:
👇 the default export: same as writing `const componentWhitelist = {...}; export default componentWhitelist`